tests/parser/tests/cases/should_pass/parenthesised_for_loop/case.hash

```diff
@@ -0,0 +1,3 @@
+(for name in names.iter() {
```

Comment:
Why should this pass?

Reply:
Treated as an expression at the end of the module.

Reply:
Okay, resolving this, see previous response.